assets/base.css

```diff
@@ -2751,11 +2751,11 @@ details[open] > .header__menu-item .icon-caret {
 
 .header__submenu .header__menu-item {
   justify-content: space-between;
-  padding: 0.8rem 2.4rem;
+  padding: 0.8rem 2rem;
 }
 
-.header__submenu .header__submenu .header__menu-item {
-  padding-left: 3.4rem;
+.header__submenu .header__menu-item:hover {
+  text-decoration: underline;
```

Comment:
Here I was mentioning `text-decoration-line: underline;`specifically. 
So `text-decoration-line` over the current `text-decoration`. 

When you use a CSS shortand property, even if you mention only 1 value, it will overwrite all the others as well. 
Text-decoration is a shortand of those properties: 
- text-decoration-color
- text-decoration-line
- text-decoration-style
- text-decoration-thickness

So even if you're just setting the underline here, you're also setting back the other properties to their default value. So the thickness we're hoping for (0.2rem) isn't applied. 

Checkout this gif that shows how the underline isn't getting thicker when hovering an active menu item: 
![](https://screenshot.click/31-34-n5vix-11xc6.gif)